Packs/ipinfo/ReleaseNotes/2_1_0.md

```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### IPinfo v2
+- ***Breaking Change:*** Updated the **brand name** in indicators extracted and relationships, to match the context path `IPinfo`.
```

Comment:
@amshamah419 wdyt?
@dorschw perhaps add the previous name so it'll be easier to keep track if anyone relied on the previous name.